src/patternfly/upgrade-examples/AlertUpgradeExamples/examples/alert-upgrade-example-pf4-alert.hbs

```diff
@@ -0,0 +1,69 @@
+{{#> alert-upgrade-examples}}
+{{#> alert alert--modifier="pf-m-success" alert--attribute='aria-label="Success Notification"'}}
+  {{#> alert-icon alert-icon--success="true"}}
+  {{/alert-icon}}
+  {{#> alert-body}}
+    {{#> alert-title}}
+      {{#> accessibility accessibility--type="sr-only"}}Success: {{/accessibility}}
+        Great Job!
+    {{/alert-title}}
+    {{#> alert-description}}
+			This is really working out
+		{{/alert-description}}
+    {{#> alert-link alert-link--modifier="pf-c-button pf-m-link"}}
```

Comment:
I don't think the links in the PF4 alert section need button styling. I think these should just be normal `<a>` links that don't need a unique class.

Reply:
I changed this to be <a> link, should I make a custom style to get rid of the underline

Reply:
These should probably just look like regular pf4 links (no text underline by default, text underline on hover) - [_base.scss](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/_base.scss#L130)

Reply:
ok looks like it takes on those styles